distributed/scheduler.py

```diff
@@ -201,11 +201,12 @@ def versions(self):
         return self._versions
 
 
+@cclass
 class WorkerState:
     """
     A simple object holding information about a worker.
 
-    .. attribute:: address
+    .. attribute:: address: str | int
```

Comment:
[From]( https://github.com/dask/distributed/pull/4302#discussion_r535324551 ) @mrocklin:

> I'm surprirsed that this can be an int. I would expect the variation to exist here for name, but address I would always expect to be a string today.

Reply:
Yeah I was surprised as well. Was running into some failures if this was restricted to `str` though. So just opted to type this as an `object` for now instead of getting too hung up on fixing the typing initially.

Reply:
Was originally trying to fix this issue along with `name` in PR ( https://github.com/dask/distributed/pull/4296 ). Wasn't sure if we wanted to fix `address` based on that discussion. As it sounds like we do, broke out that piece of the fix into PR ( https://github.com/dask/distributed/pull/4312 ).

Reply:
Ok sorry I have no clue why this was causing issues before. Have switched back to `str`. Will see how it goes 🍀